### DISCUSSION TITLE
Latching behavior for player controlled monsters

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#14535] [Ideas and suggestions]
Currently, there is no way for player controlled monsters to use "latching" behaviors, such as trying to latch onto level walls as a mantis to make use of its camouflage / ambush ability. 

I generally believe that any substantial ability an AI controlled character can perform should also be available to players when they control that character.

I've personally already started to try coding the ability to manually trigger latching and unlatching by myself, however I'm not sure if I'll manage to do it well enough to make a pull request (the biggest problem so far is networking everything properly in multiplayer which is quite a dealbreaker), so I thought I might as well suggest it in the meantime.

https://github.com/user-attachments/assets/1818b10e-9b1d-40f6-994c-3dcca86840db

**Comment by jigardi** [comment]
That's beautiful and terrifying! So have my upvote!
